api/types.proto

```diff
@@ -299,20 +324,27 @@ message UpdateStatus {
 		UPDATING = 1;
 		PAUSED = 2;
 		COMPLETED = 3;
-		// TODO(aaronl): add ROLLING_BACK, ROLLED_BACK as part of
-		// rollback support.
+		ROLLBACK_STARTED = 4;
+		ROLLBACK_PAUSED = 5; // if a rollback fails
```

Comment:
A rollback can fail if it hits the failure threshold when trying to create tasks with the previous spec. If this happens, it goes into the `ROLLBACK_PAUSED` state.

Note there's a TODO above related to this:

```
// TODO(aaronl): Should there be a separate failure threshold for
// rollbacks? Should there be a failure action for rollbacks (to allow
// them to do something other than pause when the rollback encounters
// errors)?
```